lib/postgres_to_redshift.rb

```diff
@@ -17,6 +19,8 @@ module PostgresToRedshift
   extend self
 
   def update_tables
+    PidFile.new
```

Comment:
Just looked at this library, seems like a nice api! (reminds me of a lockguard, but for the whole program).

Reply:
Indeed! Hasn't been touched for years, but still ticking

Reply:
I noticed that. If it ain't broke...